src/Symfony/Component/Notifier/Bridge/SmsFactor/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/smsfactor-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony SMSFactor Notifier Bridge",
+    "keywords": [ "sms", "smsfactor", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
+        "symfony/http-client": "^5.4|^6.0",
+        "symfony/notifier": "^5.4|^6.0",
```

Comment:
I think this does not work with notifier 5.4, must be 6.2

Reply:
How can I test this?